src/DeploymentProvider/DeploymentHelper.ts

```diff
@@ -100,29 +100,106 @@ export class DeploymentHelper {
         let uploadResponse: asa.AppsGetResourceUploadUrlResponse = await client.apps.getResourceUploadUrl(params.resourceGroupName, params.serviceName, params.appName);
         core.debug('request upload url response: ' +  JSON.stringify(uploadResponse));
         await uploadFileToSasUrl(uploadResponse.uploadUrl, fileToUpload);
+        let deploymentResource: asa.DeploymentResource = await this.buildDeploymentResource(client, params, sourceType, uploadResponse.relativePath);
+        core.debug("deploymentResource: " + JSON.stringify(deploymentResource));
+        const response = await client.deployments.beginCreateOrUpdateAndWait(params.resourceGroupName, params.serviceName, params.appName, params.deploymentName, deploymentResource);
+        core.debug('deploy response: ' + JSON.stringify(response));
+        return;
+    }
+
+    public static async deployEnterprise(client: asa.AppPlatformManagementClient, params: ActionParameters, sourceType: string, fileToUpload: string, resourceId: string) {
+        const buildServiceName = "default";
+        const buildName = `${params.appName}-${params.deploymentName}`;
```

Comment:
So for the same deployment we will only have one build no matter how many times we trigger the build. Is this expected?

Reply:
I think Yes. But one build will have multiple build results. The implementation of CLI is the same right now.